lib/mayaUsd/fileio/registryHelper.cpp

```diff
@@ -189,7 +189,6 @@ void UsdMaya_RegistryHelper::FindAndLoadMayaPlug(
                 // already loaded.
                 plug->Load();
             }
-            break;
```

Comment:
Why continue the loop if we already found and loaded the translator for the given type?